examples/tutorial/07_Geographic_Data.ipynb

```diff
@@ -113,6 +113,7 @@
    "metadata": {},
    "source": [
     "It is often useful to project the data ahead of time, particularly for large datasets, because when working interactively we often display the data many times and don't want to reproject it every time.\n",
```

Comment:
```suggestion
    "It is often useful to [project the data ahead of time](http://geoviews.org/user_guide/Projections.html#Explicitly-projecting), particularly for large datasets, because when working interactively we often display the data many times and don't want to reproject it every time.\n",
```